rectools/models/nn/sasrec.py

```diff
@@ -137,15 +205,16 @@ def __init__(
     ):
         super().__init__()
         self.n_blocks = n_blocks
-        self.multi_head_attn = nn.ModuleList(
-            [torch.nn.MultiheadAttention(n_factors, n_heads, dropout_rate, batch_first=True) for _ in range(n_blocks)]
-        )  # important: original architecture had another version of MHA
-        self.q_layer_norm = nn.ModuleList([nn.LayerNorm(n_factors) for _ in range(n_blocks)])
-        self.ff_layer_norm = nn.ModuleList([nn.LayerNorm(n_factors) for _ in range(n_blocks)])
-        self.feed_forward = nn.ModuleList(
-            [PointWiseFeedForward(n_factors, n_factors, dropout_rate, torch.nn.ReLU()) for _ in range(n_blocks)]
+        self.transformer_layers = nn.ModuleList(
```

Comment:
```suggestion
        self.transformer_blocks = nn.ModuleList(
```
Otherwise we keep same naming for different things and it's not too clear.

Reply:
fixed